products/sle/main.pm

```diff
@@ -1215,8 +1204,20 @@ elsif (get_var("Y2UITEST")) {
     # setup $serialdev permission and so on
```

Comment:
@okurz sure thing but because they never exists on o3 I can't introduce regression so I don't need prove run :) I will do this AFTER PR will be merged